cspell.json

```diff
@@ -0,0 +1,62 @@
+{
+    "version": "0.1",
+    "language": "en",
+    "enabledLanguageIds": [
+        "css",
+        "graphql",
+        "html",
+        "json",
+        "javascript",
+        "javascriptreact",
+        "markdown",
+        "mdx",
+        "plaintext",
+        "restructuredtext",
+        "sass",
+        "scss",
+        "typescript",
+        "typescriptreact",
+        "yaml",
+        "yml"
+    ],
```

Comment:
These do not seem to be required, and do not seem to be doing anything.

```suggestion
```

Reply:
It does have an effect, but not necessarily the desired effect. It adds mdx files to the list that are checked in VSCode but not when running cspell from the command line.

Reply:
k